docs/api-gateway-v2/08-01-exposesecure.md

```diff
@@ -0,0 +1,239 @@
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: httpbin
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    service:
+      name: httpbin-proxy
+      port: 8000
+      host: httpbin-proxy.kyma.local
+    rules:
+      - path: /.*
+        methods: ["GET"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["read"]
+      - path: /post
+        methods: ["POST"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["write"]
+  EOF
+  ```
+
+>**NOTE:** If you are running Kyma on Minikube, add `httpbin-proxy.kyma.local` to the entry with Minikube IP in your system's `/etc/hosts` file.
+
+The exposed service requires tokens with "read" scope for `GET` requests in the entire service and tokens with "write" scope for `POST` requests to the `/post` endpoint of the service.
+
+  </details>
+
+  <details>
+  <summary>
+  Secure a lambda function
+  </summary>
+
+1. Create a lambda function using the [supplied code](./assets/lambda.yaml):
+
+  ```
+  kubectl apply -f ./assets/lambda.yaml
+  ```
+
+2. Expose the lambda function and secure it by creating an APIRule CR:
+
+  ```
+  cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: lambda
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    service:
+      name: labmda-proxy
+      port: 8080
+      host: labmda-proxy.kyma.local
+    rules:
+      - path: /lambda
+        methods: ["GET"]
+        accessStrategies:
+          - handler: oauth2_introspection
+            config:
+              required_scope: ["read"]
+  EOF
+  ```
+
+>**NOTE:** If you are running Kyma on Minikube, add `lambda-proxy.kyma.local` to the entry with Minikube IP in your system's `/etc/hosts` file.  
+
+The exposed lambda function requires all `GET` requests to have a valid token with the "read" scope.
+
+  </details>
+</div>
+
+## Access the secured resources
+
+Follow the instructions in the tabs to call the secured service or lambda functions using the tokens issued for the client you registered.
+
+<div tabs>
+
+  <details>
+  <summary>
+  Call secured endpoints of a service
+  </summary>
+
+1. Send a `GET` request with a token with the "read" scope to the HttpBin service:
+
+  ```
+  curl -ik -X GET https://httpbin-proxy.$DOMAIN/headers -H "Authorization: Bearer $ACCESS_TOKEN_READ"
+  ```
+
+2. Send a `POST` request with a token with the "write" scope to the HttpBin's `/post` endpoint:
+
+  ```
+  curl -ik -X POST https://httpbin-proxy.$DOMAIN/post -d "test data" -H "Authorization: bearer $ACCESS_TOKEN_WRITE"
+  ```
+
+These calls return a code `200` response. If you call the service without a token, you get a code `401` response. If you call the service or its secured endpoint with a token with the wrong scope, you get the code `403` response.
```

Comment:
```suggestion
These calls return the code `200` response. If you call the service without a token, you get the code `401` response. If you call the service or its secured endpoint with a token with the wrong scope, you get the code `403` response.
```
These codes are set and their meaning doesn't change, so I'd use `the`, like you did in the last sentence.

Reply:
good point